src/main/resources/release_notes.md

```diff
@@ -35,6 +35,7 @@
 
 ### Lineup
 
+* Refactoring of lineup assistant (#163)
```

Comment:
I think it's more than refactoring the assistant, it's adding an additional feature, right?

Reply:
Changed this.